build_n_deploy/naiserator/naiserator_prod.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     team: teamfamilie
 spec:
-  image: {{ image }}
+  image: { { image } }
```

Comment:
Funker dette?

Reply:
den bare gjorde det for meg :( har ikke testa

Reply:
kan ta det bort 😆

Reply:
fixed